### DISCUSSION TITLE
Ent auth I have my backup code how do I use it

### DISCUSSION THREAD

**Discussion by ERoCx420** [ente-io/ente#3718] [Q&A]
I switched devices and I’m locked out of everything how do I use the ent auth backup phrases. I don’t see anywhere online to post this

**Comment by Damariobros** [comment]
Go to Ente.io and login. After entering your email, click Next. If email verification is enabled on your account, enter the code sent to your email and click Next. On either the two-factor, passkey, or password entry screen, there should be an option to recover your account. Click it, and you will be prompted for your recovery key. Enter your recovery key and click Next. You will be prompted to enter a new password for your account. Set a new password for your account, and from then on you will be able to log into your account using that password. If you lost access to them, disable two-factor and remove passkeys so they do not bar entry to your account.

If you have email verification enabled and are unable to receive the codes sent to your email, you will first need to contact support@ente.io to have them disable email verification on your account.

If you still have your password, you do not need to use your recovery key, unless you have lost access to your passkeys or two-factor.

It is not recommended to require Ente Auth to access Ente, and it is also not recommended to require access to other accounts like Proton or Google protected by Auth to access Ente, as this creates a chicken-and-the-egg situation which can lead to lockouts.

I hope you're able to regain access to your accounts!

**Comment by ERoCx420** [comment]
I have access to the email but I think I overwrote the tokens. When I
didn’t have access I made one stupidly not logged in. Then got a backup
code to my proton and logged in and it was the one and only password token
that I stupidly made. I’ll try this but it only shows one single password
rolling token.
